src/js/tabs/configuration.js

```diff
@@ -708,6 +708,9 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             'Japanese MSAS',
             'Indian GAGAN'
         ];
+        if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+            gpsSbas.push('NONE');
+        }
```

Comment:
Maybe we can internationalize all the options of this combo with this PR? In this way the translators will be able to translate them. Thanks!